src/reference/90-Developers-Guide/02-Coding-Guideline/00.md

```diff
@@ -175,13 +179,13 @@ object FooID {
 }
 ```
 
-#### Implicit converter for syntax (enrich-my-library) should be imported
+#### Implicit conversions for syntax (enrich-my-library pattern) should be imported
```

Comment:
No, I stand corrected. [Programming in Scala 1st ed](https://www.artima.com/pins1ed/implicit-conversions-and-parameters.html) says *implicit conversion* and the SLS actually uses a wacky term called *view*:

> Implicit parameters and methods can also define implicit conversions called views. A *view* from type `S` to type `T` is defined by an implicit value which has function type `S=>T` or `(=>S)=>T` or by a method convertible to a value of that type.

This is because the *real* implicit conversions include other things like numeric widening, which I think Paul recently tweeted about: https://twitter.com/extempore2/status/703362270835834880